src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat/ITryConvertTool.cs

```diff
@@ -15,5 +15,7 @@ public interface ITryConvertTool
         Task<bool> RunAsync(IUpgradeContext context, IProject project, CancellationToken token);
 
         string GetCommandLine(IProject project);
+
+        string? TryGetVersion();
```

Comment:
Yea, I might change it to `string? Version {get;}`. Currently there should always be a version. I used `string?`, though, because the version depends on things that could (maybe?) change. 

If the way .NET CLI tools version their host or change their file layout, we could end up not able to determine the version and I didn't want Upgrade-Assistant to break in that scenario (since try-convert version is only used for diagnostics).